### PR TITLE
Update Get-CsPolicyPackage.md

### PR DESCRIPTION
**Pull request by icchan8**

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsPolicyPackage.md [hunk 1]
```diff
@@ -40,7 +40,23 @@ Returns all policy packages available on the tenant.
 PS C:\> Get-CsPolicyPackage -Identity Education_PrimaryStudent
 ```
 
-Returns the definition of the Education_PrimaryStudent policy package, including the policies applied with the package.
+Returns only the Education_PrimaryStudent policy package.
+
+### Example 3
+```powershell
+PS C:\> $a = Get-CsPolicyPackage -Identity Education_PrimaryStudent
+PS C:\> $a.Policies.AdditionalProperties
```
Comment (dariomws) [inline teams/teams-ps/teams/Get-CsPolicyPackage.md:48]:
```suggestion
PS C:\> $a.Policies
```
I think we should remove ".AdditionalProperties":
image

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — COMMENTED** [review]
agree with @dariomws

**Review by tseward — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsPolicyPackage.md (hunk 1 above)
Comment (icchan8) [reply teams/teams-ps/teams/Get-CsPolicyPackage.md:52]: Agreed, also in the upcoming release the AdditionalProperties will need to be specified. I'll put a comment in there with a version placeholder and I'll update this PR once I know what the version number is going to be

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by icchan8** [comment]
@Kateyanne Please help merge the PR, thanks.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**